lib/system.js

```diff
@@ -1,9 +1,7 @@
 "use strict";
 
 const {Task}     = require("atom");
-const taskPath   = require.resolve("./system-task.js");
-const {op}       = require(taskPath);
-
+const {op, taskPath} = require("./system-task")
```

Comment:
Please use semicolons.

Reply:
Fixed in https://github.com/file-icons/fs/pull/8/commits/5c1f55f858343866a123c1e5a211486d050f2522